### DISCUSSION TITLE
Add back the moderation bat icon to the main menu

### DISCUSSION THREAD

**Discussion by Hiviexd** [ppy/osu#23205] [Ideas]
in osu!stable, moderators have this cute little bat icon in the bottom right of the main menu

![](https://nats.are-la.me/6ffpboh.png)
![](https://nats.are-la.me/6VX3ZWx.png)

i speak on behalf of the GMT/NAT that we want this icon to make a return to the main menu of lazer.

**Comment by chromb** [comment]
as an alternate i would like to propose including an image of the entire timezone of Greenwich Mean Time.
